spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -504,6 +506,65 @@ def set_color_scheme(self, color_scheme, reset=True):
         except AttributeError:
             pass
 
+    def set_use_pager(self, use_pager):

```

Comment:
Let's rename this to `set_paging`.

Reply:
Also, since all these methods apply at the `shellwidget` level, I think we should remove them here, leave them only in `shellwidget` and call them like this

    sw = client.shellwidget
    sw.set_paging

(and the same for all other methods) from the plugin.